node/test/unit/tessel.js

```diff
@@ -1337,10 +1353,10 @@ exports['Tessel.Pin'] = {
       test.equal(this.b.pin[pinIndex].interruptMode, 'change');
 
       // Simulate receipt of pin state changes
-      this.a.sock.read.returns(new Buffer([REPLY.ASYNC_PIN_CHANGE_N + pinIndex]));
+      this.a.sock.read.returns(new Buffer([(REPLY.ASYNC_PIN_CHANGE_N + pinIndex) | (0 << 3)]));
       this.a.sock.emit('readable');
 
-      this.b.sock.read.returns(new Buffer([REPLY.ASYNC_PIN_CHANGE_N + pinIndex]));
+      this.b.sock.read.returns(new Buffer([(REPLY.ASYNC_PIN_CHANGE_N + pinIndex) | (0 << 3)]));
```

Comment:
Oh, I think I understand the comment you made about OR'ing with 0... If I was trying to be explicit in showing it clearing the bit I think it should have been:

`(REPLY.ASYNC_PIN_CHANGE_N + pinIndex) & ~(1 << 3)`